stats/datadog/collector_test.go

```diff
@@ -4,16 +4,15 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/loadimpact/k6/lib"
 	"github.com/loadimpact/k6/stats"
 	"github.com/loadimpact/k6/stats/statsd/common"
 	"github.com/loadimpact/k6/stats/statsd/common/testutil"
 	"github.com/stretchr/testify/require"
 )
 
 func TestCollector(t *testing.T) {
-	var tagSet = lib.GetTagSet("tag1", "tag2")
-	var handler = tagHandler(tagSet)
+	var tagSet = stats.ToSystemTagSet([]string{stats.TagProto.String(), stats.TagGroup.String()})
```

Comment:
This seems like a regression to me. Does this mean that you won't be able to blacklist not systemtags ?

Reply:
@MStoykov what "won't be able to blacklist not systemtags" mean?

Reply:
I mean that you changed the tags above from not systemTags to systemTags, because you can't blacklist systemTags after your change

Reply:
@MStoykov it's still there https://github.com/loadimpact/k6/pull/1148/files/6ec23ab5e93950d47cc1c6bcaebe31e1b126d570#diff-2861003ad31b1de3cce4af17e27abff6R15